src/timetable/timetable-row/row-bar.tsx

```diff
@@ -59,7 +59,15 @@ export const RowBar = ({
             });
         }
 
-        rowCells.forEach((cell, index) => {
+        const sortedRowCells = rowCells.sort((a, b) => {
```

Comment:
to avoid mutating the props

```suggestion
        const sortedRowCells = [...rowCells].sort((a, b) => {
```